frontend/package.json

```diff
@@ -9,7 +9,8 @@
     "lint:fix": "eslint ./src/** ./tests/** --fix",
     "format": "prettier --write .",
     "test": "vitest",
-    "coverage": "vitest run --coverage"
+    "coverage": "vitest run --coverage",
+    "license-checker": "license-checker --excludePackage \"ksummarized@0.1.0\" --summary --onlyAllow \"MIT;ISC;Apache-2.0;BSD-3-Clause;BSD-2-Clause;CC0-1.0;0BSD;Python-2.0;MPL-2.0;CC-BY-4.0;CC-BY-3.0\""
```

Comment:
Is there a way to pull the `--onlyAllow` into a variable or a separate file?

Reply:
I moved it to a separate file.